### PR TITLE
Impl/add additional video streaming capabilities param

### PR DESCRIPTION
**Pull request by YarikMamykin**
Added `additionalVideoStreamingCapabilities` param into `VideoStreamingCapability` structure.
Update InterfaceGenerator to support recursive structures generation from API.

### CODE REVIEW

File: InterfaceParser/parsers/rpc_base.py [hunk 1]
```diff
@@ -229,6 +231,8 @@ def _parse_struct(self, element, prefix):
                 raise ParseError("Unexpected subelement '{}' in struct '{}'".format(subelement.name, params["name"]))
         params["members"] = members
 
+        self._types.pop(struct.name, None)
```
Comment (LuxoftAKutsan) [inline InterfaceParser/parsers/rpc_base.py:249]: Please add appropriate comments, why we need to remove from structs

**Review by LuxoftAKutsan — APPROVED** [review]

File: InterfaceParser/parsers/rpc_base.py (hunk 1 above)
Comment (LitvinenkoIra) [reply InterfaceParser/parsers/rpc_base.py:249]: @YarikMamykin, please create a good commit message with the explanation of what was fixed and how

**Review by LitvinenkoIra — COMMENTED** [review]
please check https://github.com/LuxoftSDL/rpc_spec/pull/3#discussion_r452035923

**Review by IGapchuk — APPROVED** [review]

File: InterfaceParser/parsers/rpc_base.py (hunk 1 above)
Comment (YarikMamykin) [reply InterfaceParser/parsers/rpc_base.py:249]: @LuxoftAKutsan @LitvinenkoIra fixed in https://github.com/LuxoftSDL/rpc_spec/pull/3/commits/15d6abfb7656b808e59feade4b53d51b032dde3f

**Review by LitvinenkoIra — APPROVED** [review]

**Pull request merged**